### ISSUE TITLE
 --without-file-storage-dir is broken for 2.7

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#448]
Originally reported of Google Code with ID 392.

```
./configure --without-file-storage-dir
make
...
Traceback (most recent call last):
  File "/home/ultrotter/source/ganeti/ganeti/./autotools/convert-constants", line 30, in <module>
    from ganeti import constants
  File "/tmp/gntbuild.Gc6UjTYU/ganeti/constants.py", line 30, in <module>
    from ganeti import pathutils
  File "/tmp/gntbuild.Gc6UjTYU/ganeti/pathutils.py", line 32, in <module>
    DEFAULT_FILE_STORAGE_DIR = vcluster.AddNodePrefix(_autoconf.FILE_STORAGE_DIR)
  File "/tmp/gntbuild.Gc6UjTYU/ganeti/vcluster.py", line 194, in AddNodePrefix
    assert os.path.isabs(path)
AssertionError
make: *** [src/Ganeti/Constants.hs] Error 1
make: *** Deleting file `src/Ganeti/Constants.hs'

```
Originally added on 2013-03-01 21:07:54 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-03-04 10:41:32 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Fixed via multiple patches. I would guess the QA/unittests could be broken in 2.6 as well, but we never tried to run them without file storage.

In any case, actually building without file storage is possible now.
```
Originally added on 2013-03-04 15:34:29 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Marking 2.7 bugs fixed, as 2.7.0~rc1 was released, and should be ready to be tested in production environments.
```
Originally added on 2013-05-09 05:42:48 +0000 UTC.

Changed State: Released
